.github/workflows/vulnerability-scanning-on-repo.yml

```diff
@@ -0,0 +1,95 @@
+name: Vulnerability Scanning on Repo
+on:
+  workflow_call:
+    inputs:
+      APP_NAME:
```

Comment:
low-importance idea: 
(I won't normally bother, but it will be so painful to change this later)
Maybe REPO_NAME is more correct (So IF one day we stop attaching to gh releases THEN we can easily map to repo)?

```
use: telicent-oss/shared-workflows/.github/workflows/vulnerability-scanning-on-repo.yml
 with:
   REPO_NAME:  ${{ github.repository }} # format: org/repo-name
```

Reply:
Is this assuming the repo name will always match the app name?

Reply:
I think u are correct